packages/admin-ui/src/pages/rollups/components/Zkevm.tsx

```diff
@@ -0,0 +1,212 @@
+import React from "react";
+import { api, useApi } from "api";
+import { AppContext } from "App";
+import Card from "components/Card";
+import ErrorView from "components/ErrorView";
+import Loading from "components/Loading";
+import Col from "react-bootstrap/Col";
+import Row from "react-bootstrap/Row";
+import SubTitle from "components/SubTitle";
+import ExecutionClient from "./columns/ExecutionClient";
+import LegacyGeth from "./columns/LegacyGeth";
+import OptimismNode from "./columns/OptimismNode";
+import { useOptimismConfig } from "./useOptimismConfig";
+import "./columns.scss";
+import { Alert, Button, Form } from "react-bootstrap";
+import Input from "components/Input";
+import { confirm } from "components/ConfirmDialog";
+import { disclaimer } from "../data";
+import { withToast } from "components/toast/Toast";
+import ZkevmCommunity from "./ZKevmCommunity";
+
+export default function Zkevm({ description }: { description: string }) {
```

Comment:
Here you should:
1. Know whether the package is installed or not
2. If the package is installed, show it as selected if it is running and unselected if not
3. If the package is not installed, show it as unselected
4. If the user unselects the package and hits "apply", the package should be stopped
5. If the user selects the package and hits "apply" the package should be (installed) + started
6. If the package is installed and running, a button should be shown inside the card ("Go to UI")

To install the package you need to know its identifier, which is its `dnpName`. In this case, the `dnpName` of the package will be: `zkevm-tokens-withdrawal.dnp.dappnode.eth`